GamingAnthems/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "TechLewi",
+    "id": "590575157718941708"
+  },
+  "service": "GamingAnthems",
+  "description": {
+    "en": "No1 Gaming Radio Station"
+  },
+  "url": [
+    "gaminganthems.com",
+    "raidstop.co.uk",
+    "raidstop.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://imgur.com/9PEupja",
+  "thumbnail": "https://imgur.com/j7u9H9X",

```

Comment:
still not the i.imgur links with ending .png / .jpg

Reply:
it still works tho
ill change it tho

Reply:
these links wont work on your website, they need to be the direct image links